crates/rolldown_error/src/events/resolve_error.rs

```diff
@@ -13,17 +13,27 @@ pub struct DiagnosableResolveError {
   pub help: Option<String>,
 }
 
+impl DiagnosableResolveError {
+  fn importee_str(&self) -> &str {
+    let s = match &self.importee {
+      DiagnosableArcstr::String(str) => str.as_str(),
+      DiagnosableArcstr::Span(span) => &self.source.as_str()[*span],
+    };
+    &s[1..s.len() - 1]
```

Comment:
Quote was coming from the source itself, so I stripped it here and added back as single quote in `format!` below. This made a bit noisy diff in snapshots, but choosing double quotes would also make more diffs.